osu.Game/Overlays/Profile/Sections/Ranks/DrawableScore.cs

```diff
@@ -159,5 +135,61 @@ protected override IEnumerable<Vector2> ComputeLayoutPositions()
                     yield return new Vector2(DrawWidth * i * (count == 1 ? 0 : 1f / (count - 1)), 0);
             }
         }
+
+        public class PPScore : DrawableScore

```

Comment:
let's move these two to their own classes

Reply:
I intentionally placed them there so I could make the constructor private. That also avoided the problem of having to set other fields not private